interpreter/binary/decode.ml

```diff
@@ -96,7 +96,7 @@ let rec vsN n s =
   then (if b land 0x40 = 0 then x else Int64.(logor x (logxor (-1L) 0x7fL)))
   else Int64.(logor x (shift_left (vsN (n - 7) s) 7))
 
-let vu1 s = Int64.to_int (vuN 1 s)
+let vu2 s = Int64.to_int (vuN 2 s)
```

Comment:
Perhaps we should just eliminate this and use u8 below.

Reply:
Could do, but then would have to check that high bits are zero in callers, which is a little inconvenient.

Reply:
Done.